apps/meteor/client/lib/onToggledFeature.ts

```diff
@@ -15,7 +15,7 @@ export const onToggledFeature = (
 	},
 ): (() => void) => {
 	const observer = new QueryObserver(queryClient, {
-		queryKey: ['ee.features'],
```

Comment:
A quick search shows me that this query key is also being used by [`hasLicense`](https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/license/client/index.ts). Could you check and see if it's not relevant to change there as well?

Reply:
@aleksandernsilva  Yeah this function is only used here https://github.com/RocketChat/Rocket.Chat/blob/fix/ee-query/apps/meteor/app/livechat-enterprise/client/index.ts#L3

This function mainly sets the omnichannel EE items on the omnichannel admin section, I've done some testing and there were no difference in the way these items are behaving. So we should be ok